packages/fxa-settings/src/components/Settings/AppLayout/index.tsx

```diff
@@ -18,6 +20,7 @@ export const AppLayout = ({ children }: AppLayoutProps) => {
       className="flex flex-col justify-between min-h-screen"
       data-testid="app"
     >
+      {createPortal(<BrandMessaging viewName="settings" />, document.body)}
```

Comment:
Nice, one question I had was if this was going to be accessible. (Haven't tested locally but portal is a good call.)

Reply:
Portal let the fixed positioning work nicely and 'break out' of the components container.